src/PostgREST/App.hs

```diff
@@ -280,10 +280,10 @@ augumentRequestWithJoin schema allRels request =
 
 buildReadRequest :: [Relation] -> ApiRequest -> Either Text ReadRequest
 buildReadRequest allRels apiRequest  =
-  augumentRequestWithJoin schema rels =<< first formatParserError (foldr addFilter <$> (addOrder <$> readRequest <*> ord) <*> flts)
+  augumentRequestWithJoin schema rels
+  =<< first formatParserError (foldr addFilter <$> (foldr addOrder <$> readRequest <*> ords) <*> flts)
```

Comment:
but yes, that line i guess could be written in a nicer way if ai knew any haskell :)
But i think it's clear enough what's happening on that line so maybe let's leave the refactoring for some other day since any way i see some patterns emerging that would allow some overall refactoring.